code/controllers/subsystem/discord.dm

```diff
@@ -46,6 +46,7 @@ SUBSYSTEM_DEF(discord)
 
 /datum/controller/subsystem/discord/Initialize(start_timeofday)
 	// Check for if we are using TGS, otherwise return and disables firing
+	UNTIL(GLOB.tgs_initialized)

```

Comment:
How is this even possible ? Doesn't tgs initialize before master inits ? None of these init procs return control ?

Reply:
This all stems from the fact that `TgsNew()` can now sleep and `/world/New` is
implicitly `set waitfor = FALSE`.

This may being excessive here, but it should just compile to an if